src/index.ts

```diff
@@ -1,2 +1,3 @@
-export * from './dsl/index';
-export * from './model/index';
+export * as oas30 from "./oas30";
+export * as oas31 from "./oas31";
+export { Server, ServerVariable } from "./model/server"
```

Comment:
we can also have export of specification extension in the root if needed.
currently they are inside each of `oas30` and `oas31` only